section2-week1/exercise7/Main.cpp

```diff
@@ -11,93 +11,96 @@ using namespace std;
 
 int main()
 {
-	int value, random, answer;
-	ans = 0;
+	int value, random;
 
-	srand(time(NULL));
-	random = rand() % (1 - 4);
 
 	cout << "Hey you, let's play Rock, Paper, Scissors!" << endl;
-
 	cout << "1 - Rock\n2 - Paper\n3 - Scissors" << endl;
 	cout << "choose you weapon: " << endl;
 
-	cin >> value;
+	enum Weapon
+	{
+		Rock, Paper, Scissors
+	};
+	srand(time(NULL));
+	random = rand() % (1 - 4);
+	Weapon weapon = (Weapon)random;
 
-	cout << "\nI have selected the weapon: " << random << endl;
+	cin >> value;
 
 
+	enum Answers
+	{
+		tie, YouWin, YouLose
+	};
+	Answers answer = tie;
+	
 	//comparing the weapons
-	switch (value) 
+	switch (weapon)
 	{
-		case 1:
-			cout << "You have selected Rock" << endl;
-			if (random == 1) 
-			{
-				answer = 1; ;
-			}
-			if (random == 2)
-			{
-				answer = 3;
-			}
-			if (random == 3)
-			{
-				answer = 2;
-			}
-			break;
-		case 2:
-			cout << "You have selected paper" << endl;
-			if (random == 1)
-			{
-				answer = 2;
-			}
-			if (random == 2)
-			{
-				answer = 1;
-			}
-			if (random == 3)
-			{
-				answer = 3;
-			}
-			break;
-		case 3:
-			cout << "You have selected Scissors" << endl;
-			if (random == 1)
-			{
-				answer = 2;
-			}
-			if (random == 2)
-			{
-				answer = 3;
-			}
-			if (random == 3)
-			{
-				answer = 1;
-			}
-			break;
-		default:
-			cout << "You do not know how to play this game...";
-
+	case Rock:
+		cout << "I have selected Rock" << endl;
+		if (value == 1)
```

Comment:
The variable `value` could be directly compared to the enum values, instead of the numbers. You could do `if (value == rock)` for instance. Also, remember to use `else` and `else if` when the values are mutually exclusive. Can `value` ever be 1 AND 2 at the same time?